app/models/models.js

```diff
@@ -2196,20 +2191,32 @@ YUI.add('juju-models', function(Y) {
      * @return {Object} export object suitable for serialization.
      */
     exportDeployer: function() {
-      var serviceList = this.services,
-          relationList = this.relations,
-          defaultSeries = this.environment.get('defaultSeries'),
-          result = {
-            envExport: {
-              services: {},
-              relations: []
-            }
-          };
+      var defaultSeries = this.environment.get('defaultSeries'),
```

Comment:
ok added a card as a follow up. It's a bit past time we moved onto last year's release.